src/parallel/CollectiveCommunicationSingleNonBlocking.h

```diff
@@ -87,7 +83,7 @@ class CollectiveCommunicationSingleNonBlocking: public CollectiveCommunication {
 	}
 
 	// documentation in base class
```

Comment:
`@copydoc` ?

Reply:
no need for copydoc, this _should_ be found automatically